lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -124,6 +214,16 @@ class tethys::ScienceSensorsSystemPrivate
   /// in the ENU world frame.
   public: std::vector<pcl::PointCloud<pcl::PointXYZ>::Ptr> timeSpaceCoords;
 
+  /// \brief Resolution of spatial coordinates in meters in octree, for data
+  /// search.
+  public: float spatialRes = 0.1f;
```

Comment:
```suggestion
  // TODO(tfoote) Magic Number that should be parameterized.
  public: float spatialRes = 0.1f;
```

Reply:
Fixed in #110 and now merged into this PR in 0af6a57